src/components/attendee/AttendeeLayout/AttendeeLayout.module.scss

```diff
@@ -53,9 +54,10 @@ $-z-index--popover: layer.level($-z-index--ui);
   }
 }
 
-.LayerUiAbsolute {
-  position: absolute;
-  top: 0px;
+.LayerUiRelative {
```

Comment:
Wait, are we renaming this now? How about `layer`? Do we need it complicated?

Reply:
renamed into `relative`